auth/src/main/res/layout/register_email_layout.xml

```diff
@@ -14,23 +14,23 @@
             is showing we use a relative layout to position them absolutely
         -->
         <android.support.design.widget.TextInputLayout
-            xmlns:app="http://schemas.android.com/apk/res-auto"
             android:id="@+id/email_layout"
```

Comment:
See comment above, we could lock the email field from here: `android:enabled="false"`.